src/libmeasurement_kit/dns/resolver.hpp

```diff
@@ -0,0 +1,152 @@
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
```

Comment:
More nitpicking: I presume it would be more obvious if this file was called `system_resolver.hpp`